src/app/providers/data-store/data-store.ts

```diff
@@ -189,18 +209,24 @@ export class DataStoreProvider {
     });
   }
 
+  /**
+   * Deletes the old index db databases.
+   */
   async clearIndexedDB() {
     try {
       // Clear all IndexedDB databases
       (await window.indexedDB.databases()).forEach((db) => {
         window.indexedDB.deleteDatabase(db.name);
       });
     } catch (err) {
-      this.reportLog('clearIndexedDB', '', err, LogType.ERROR);
+      this.reportLog('clearIndexedDB', '', err);
```

Comment:
if your updating error logs elsewhere in this file, can this to be consistant.

Eg youve added error here:
this.reportLog('initDataStore error', '', err);